core/src/cli.rs

```diff
@@ -10,6 +11,21 @@ struct CliArgs {
     config_file: Option<String>,
 }
 
```

Comment:
hmm why is this in cli.rs?

Reply:
Hmm, fair point. I see it as the counterpart to `init_cli` -- `init_cli` is run at the beginning of the `async main` and `wait_for_interrupt` is run at the end. But maybe `cli` isn't the best name for this module any more? Maybe `process` or something?